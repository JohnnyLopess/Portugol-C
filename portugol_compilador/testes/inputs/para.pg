programa  
{  
    funcao inicio(){
        inteiro tab  
  
        para (inteiro c=1; c<=10; c++){ 
            tab=c*3  
            escreva ("operação feita")  
        }  
    }  
}  