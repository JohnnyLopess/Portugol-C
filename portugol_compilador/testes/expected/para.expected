#include <stdio.h>

int main() {
    int tab;
    for (int c = 1; c <= 10; c++) {
        tab = c * 3;
        printf("operação feita");
    }
    return 0;
}